lib/chef/knife/bootstrap/templates/README.md

```diff
@@ -1,12 +1,11 @@
 This directory contains bootstrap templates which can be used with the -d flag
 to 'knife bootstrap' to install Chef in different ways. To simplify installation,
 and reduce the matrix of common installation patterns to support, we have
-standardized on the [Omnibus](https://github.com/opscode/omnibus-ruby) built installation 
+standardized on the [Omnibus](https://github.com/chef/omnibus) built installation
 packages.
 
 The 'chef-full' template downloads a script which is used to determine the correct
-Omnibus package for this system from the [Omnitruck](https://github.com/opscode/opscode-omnitruck) API. All other templates in this directory are deprecated and will be removed
-in the future.
+Omnibus package for this system from the [Omnitruck](https://github.com/chef/omnitruck) API.
```

Comment:
would be better to link to the API docs here? https://docs.chef.io/api_omnitruck.html